client/src/containers/MusicianProfileContainer.js

```diff
@@ -0,0 +1,26 @@
+import MusicianProfile from '../components/MusicianProfile';
+import { updateUser } from '../actions/userActions';
+import { getUserId } from '../reducers/userReducer';
+import gimmeData from '../utils/gimmeData';
+import { getModels } from '../reducers/modelsReducer';
+
+function urlFn(state) {
+  return `users/${getUserId(state)}`;
+}
+
+const mapDispatchToProps = {
+  updateUser
+};
+
+function getInitialValues(state) {
```

Comment:
I believe this is the correct way to supply initial values for form defaults to redux form.